src/main/java/com/robothand/highqualitybot/Bot.java

```diff
@@ -15,25 +15,17 @@
  */
 public class Bot {
     static JDA api;
-    public static String PREFIX;
+    public static Config config;
     public static final String VERSION = "0.1";
 
     public static void main(String[] args) {
-        File config;
-        String token;
+        config = null;
```

Comment:
Ah, forgot to take that out. Threw it in to quell a "might not have been initialized" when member methods were used outside of the try block, but that's no longer an issue